src/content/_data/events.yaml

```diff
@@ -99,10 +99,10 @@ make-app:
   teamSize: 2
   lang: cpp
   rounds:
-    - Elimination round (Multiple-choice based questions based on concepts of Java/Android).
-    - |-
-      The qualifying teams would be asked to develop an android/ios application overnight at their own places and next day have to present their project to judges for final evaluation
+    -  The teams would be asked to develop an android/ios application on spot at day 2 of Sankalan. Theme will be provided on spot.They will be given 8 hours that is from 10am-6pm. They have to present their project to judges for final evaluation
```

Comment:
just keep first sentence in rounds.
and move rest of them to details, as separate points.

Reply:
okk